quarkus-modules/quarkus-testcontainers/customer-service/pom.xml

```
@@ -0,0 +1,164 @@
+<?xml version="1.0"?>
+<project xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 https://maven.apache.org/xsd/maven-4.0.0.xsd"
+         xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance">
+    <modelVersion>4.0.0</modelVersion>
+    <groupId>info.customer</groupId>
+    <artifactId>customer-service</artifactId>
+    <version>1.0.0-SNAPSHOT</version>
```

We need to add the parent module for our CI build to work properly:
```suggestion
    <modelVersion>4.0.0</modelVersion>
    <groupId>info.customer</groupId>
    <artifactId>customer-service</artifactId>
    <version>1.0.0-SNAPSHOT</version>

    <parent>
        <groupId>com.baeldung</groupId>
        <artifactId>quarkus-testcontainers</artifactId>
        <version>1.0.0-SNAPSHOT</version>
    </parent>
```